### ISSUE TITLE
How can I un-hide all node_module functions from the stack?

### ISSUE THREAD

**Issue by jackquack** [node-inspector/node-inspector#652]
I can't seen to get node_modules to show up in the stack.

I'm trying to debug a node_module that I'm having trouble with, but all the other node module functions in the stack are hidden from me, so my debugging is very contrained.

I've attacked an image of my stack, and a screenshot of as far back in the stack I can go.
![screen shot 2015-05-22 at 2 33 13 pm](https://cloud.githubusercontent.com/assets/554881/7777265/a7257fa6-008f-11e5-8040-3c0adab7de2d.png)
![screen shot 2015-05-22 at 2 33 18 pm](https://cloud.githubusercontent.com/assets/554881/7777266/a731f8da-008f-11e5-8604-b3ae40f34064.png)

I have tried running with ```node-debug --cli --hidden='[]' server.js``` but that didn't do it...

**Comment by 3y3** [comment]
I think you work in directory without package.json? Is it right?

**Comment by jackquack** [comment]
> I think you work in directory without package.json? Is it right?

Nope. I'm running a regular node.js application, complete with a regular package.json file. In the screenshot, I'm inside one of my node_modules (node-openid to be specific).

**Comment by 3y3** [comment]
Hm... Maybe problem in placed in localStorage? Can you clear localStorage for Node Inspector tab?

**Comment by jackquack** [comment]
Well, I think that fixed it for me. Alright then, thanks :D

**Comment by jackquack** [comment]
I think there might be an issue when the same instance of Chrome is used multiple times. I didn't like opening and closing chrome each time, so i just left it open, used the ```--cli``` option and refreshed each time I restarted node-inspector. But then this issue occurs. So now I've given in and close the browser each time I restart the debugger. Would it be helpful if I posted the contents of my localStorage when the issue occurs?
